roles/slurm/tasks/management.yml

```diff
@@ -179,6 +179,17 @@
   tags: 'slurm.conf'
   become: true
 
+- name: 'Install Slurm GRES config file.'
```

Comment:
Same as for the client: should we remove the `when`?

Reply:
Done.